inference-engine/thirdparty/clDNN/src/graph_optimizer/remove_redundant_reorders.cpp

```diff
@@ -232,6 +232,9 @@ void remove_redundant_reorders::run(program& p) {
         if (!ident.second)
             continue;
 
+        if (r_node.is_output() && i_layout.format != o_layout.format)
+            continue;
+
```

Comment:
If a reorder is an output node with a size (1,18,1,1). And only format is different (satisfy the condition of 1d along feature side and dense) the target reorder can be removed. I think it needs to add exception logic for this case.